src/main/java/com/bootcamp/demo/FirebaseInitializer.java

```diff
@@ -0,0 +1,28 @@
+package com.bootcamp.demo;
+
+import com.google.firebase.FirebaseOptions;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+import static com.google.auth.oauth2.GoogleCredentials.fromStream;
+import static com.google.firebase.FirebaseApp.initializeApp;
+import static java.lang.System.getProperty;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Service
+public class FirebaseInitializer {
+
+    @PostConstruct
+    public void init() throws IOException {
+        final var serviceAccount = new ByteArrayInputStream(getProperty("firebaseKey").getBytes(UTF_8));

```

Comment:
Just as a check, this is due to the version used from mater/other branches, right? Same for `FirebaseConfig`.